src/app/containers/CompanyForm.js

```diff
@@ -1,107 +1,61 @@
 import React, { Component } from 'react';
-
-import FormGroup from '@material-ui/core/FormGroup';
-import TextField from '@material-ui/core/TextField';
 import Button from '@material-ui/core/Button';
-import EventIcon from '@material-ui/icons/Event';
-import FormControl from '@material-ui/core/FormControl';
-import FormLabel from '@material-ui/core/FormLabel';
-import FormControlLabel from '@material-ui/core/FormControlLabel';
-import Checkbox from '@material-ui/core/Checkbox';
-
-/* eslint-disable */
-import { MuiPickersUtilsProvider } from 'material-ui-pickers';
-// pick utils
-import DateFnsUtils from '@date-io/date-fns';
-import { DatePicker } from 'material-ui-pickers';
-
-import { checklistData } from '../../assets/checklistData';
-
-export class CompanyForm extends Component {
+import { Field, reduxForm } from 'redux-form';
+
+import CompanyName from '../components/form/CompanyName';
+import InterviewDate from '../components/form/InterviewDate';
+import CompanyChecklistWrapper from '../components/form/CompanyChecklistWrapper';
+import submitCompanyForm from '../store/actions/submitCompanyForm';
+import checklistData from '../../assets/checklistData';
+
+const validate = values => {
+  const errors = {};
+  const requiredFields = ['companyName', 'interviewDate'];
+  requiredFields.forEach(field => {
+    if (!values[field]) {
+      errors[field] = 'Required';
```

Comment:
Do you think it might make sense to move this into a separate file called for Validations? This way all the validation functions are in the same file?